src/client/context.rs

```diff
@@ -672,6 +693,18 @@ impl Context {
         self.send_message(target_id.into(), |m| m.content(content))
     }
 
+    /// Allows to configure channel options like position, name, etc.
+    /// You can see available methods in [`EditChannel`] docs.
```

Comment:
If we're not linking to `EditChannel` here (since it's in the function signature), it can just be written as "\`EditChannel`\" without the brackets, to avoid rustdoc from trying to create it into a link.